deploy/eck-integrations/values.yaml

```diff
@@ -0,0 +1,493 @@
+## @section elasticsearchRef define outputs of elasticsearch type and connection parameters
+elasticsearchRefs:
+  ## the key is the outputName
+  ## @param elasticsearchRefs.outputNameΧ.namespace [optional]
+  ## @param elasticsearchRefs.outputNameΧ.name
+  ##        The name associating an Elastic resource not managed by the ECK operator.
+  ## @param elasticsearchRefs.outputNameΧ.secretName
+  ##        secretName is the name of an existing Kubernetes secret that contains connection information for associating
+  ##        an Elastic resource not managed by the eck-operator. The referenced secret must contain the following:
+  ##        - `url`: the URL to reach the Elastic resource.
+  ##        - `username`: the username of the user to be authenticated to the Elastic resource.
+  ##        - `password`: the password of the user to be authenticated to the Elastic resource.
+  ##        - `ca.crt`: the CA certificate in PEM format (optional).
+  ##        - `api-key`: the key to authenticate against the Elastic resource instead of a username and password
+  ##                     (supported only for `elasticsearchRefs` in AgentSpec). This field cannot be used in
+  ##                     combination with the other fields name, namespace or serviceName.
+  ## E.g.
+  ## default:
+  ##   secretName: es-ref-secret [for *non* eck-managed elasticsearch]
+  ##   name: es-ref-name [for eck-managed elasticsearch]
+  ##   namespace: es-ref-namespace [for eck-managed elasticsearch]
```

Comment:
Should we have a default here? or at least an empty map so that the template renders?

Reply:
to be honest I was aiming for the template not to render if this is empty 😁 this is kinda crucial for defining the output and only the user knows the details of it. But if you have a better default idea here more than happy to incorporate it